test/kdump_test.rb

```diff
@@ -729,4 +729,29 @@
       expect(memory).to eq(high: "64", low: "32")
     end
   end
+
+  describe ".AutoYaST" do
+    before do
+      Yast::Mode.SetMode(mode)
+    end
+
+    context "during profile import" do
+      let(:mode) { "autoinstallation" }
+      let(:profile) { {"add_crash_kernel"=>true,
+                        "crash_kernel"=>"256M",
+                        "general"=>{"KDUMP_SAVEDIR"=>"file:///var/dummy"}
+                      }
+                    }
+      # bnc#995750
+      it "imported values will not be overwritten by the proposal" do
+        Yast::Kdump.Import(profile)
+        Yast::Kdump.Propose
+        ret = Yast::Kdump.Export
```

Comment:
I see you point. For "add_crash_kernel" and "KDUMP_SAVEDIR" it would be possible but for "crash_kernel" there is no interface. But I would like to test "crash_kernel" because this value has been overwritten by Proposal in the bug too.